api-put-object.go

```diff
@@ -205,10 +205,13 @@ func (c Client) putObjectSingle(bucketName, objectName string, reader io.Reader,
 		hashAlgos["sha256"] = sha256.New()
 	}
 
-	if size <= minPartSize {
+	if size <= minPutBufferSize {
```

Comment:
This was minPartSize, it means else was never entered, it could have buggy code.